app/Http/Controllers/Frontend/Admin/EventController.php

```diff
@@ -49,6 +49,7 @@ public function renderSuggestions(): View {
         return view('admin.events.suggestions', [
             'suggestions' => EventSuggestion::where('processed', false)
                                             ->where('end', '>', DB::raw('CURRENT_TIMESTAMP'))
+                                            ->where('user_id', '!=', auth()->user()->id) //moderator can't accept their own suggestions
```

Comment:
But I as an admin want to accept my own suggestions.

Reply:
You as a admin can just create events. Why would you suggest them? 🤔

Reply:
I don't like the admin Panel's creation template. 🙃

Reply:
But it's the same as when you accept suggestions. 😅